docker-compose.yml

```diff
@@ -1,42 +1,21 @@
-version: '3'
-services:
-  kafka:
```

Comment:
I don't believe we should be removing kafka from here - our other services rely on it to be there including in the "depends_on" section of several of them.

Reply:
Yeah, we aren't really getting rid of kafka just moving it from the ODE to the [jpo-utils docker compose](https://github.com/usdot-jpo-ode/jpo-utils/blob/develop/docker-compose-kafka.yml).

Reply:
So if I run either the make command or just `docker compose up -d` from here, will kafka be ran? We've got several of these services with "depends_on" set to kafka but with this removal we don't have a known kafka service for them to depend on do we?

Reply:
Yes, as long as the env variable `COMPOSE_PROFILE` includes the kafka service.

Reply:
I can verify that the dependency relationships function when referencing external docker-compose files.